### DISCUSSION TITLE
InAppConsume is called incorrectly - SDK bug?

### DISCUSSION THREAD

**Discussion by urbankelvin** [Iterable/react-native-sdk#472] [General]
HI guys, we have discovered that for some reasons, this InAppConsume is always called when we initiated the app no matter what we do. Is that a bug? 
![image (23)](https://user-images.githubusercontent.com/102268370/220480460-26d39b71-6106-4db6-9309-0cd61a1bfbdb.png)
![image (22)](https://user-images.githubusercontent.com/102268370/220480465-fb3fe473-4b23-480f-826e-f85e19560219.png)

**Comment by lposen** [comment]
Hi @urbankelvin,

I'm cleaning up stale github discussion and came across this.

Is this still occuring?  If so, please let me know the version of the SDK that you are using.
